script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -330,7 +335,7 @@ Thinking about it, I've been gazing at this scenery for almost a year and a half
 [sha:bafcd299c8a7ca91aebca62de8cf9fa7b7454847]{
 -- Page 24, Offset 1210.
 -- 電車での登校風景もこれっきりだ。
-This is the last time I will see this view on my train ride to school.
+This will be the last time I see this view I've been so accustomed to when going to school.
```

Comment:
I agree with the 'will be' change but not the 'been so accustomed to' part. It bloats the sentence and isn't in the jp either. I think the implication that 'he's used to this view' is already very evident to the reader from the surrounding lines and does not need further clarification.